app/git/git_test.go

```diff
@@ -311,3 +311,15 @@ func TestNewReleaseNonExistingrepo(t *testing.T) {
 		t.Fatalf("NewRelease did not return any error when trying to add a release to a non-existing repo")
 	}
 }
+
+func TestValidateUserNonExisting(t *testing.T) {
+	if err := ValidateUser("thisuserdoesnotexist_42424242424242424242424000"); err == nil {
```

Comment:
_Let me just create this user :stuck_out_tongue_winking_eye:_

Reply:
> _Let me just create this user 😜_

go for it ;p

Reply:
I pretty much mean the entire file. (All functions which are interacting directly with GitHub.) Unit tests should not have any external dependencies (GitHub). Integration tests are more extensive and also would show whether different systems (Shortinette and GitHub) are working together.

Reply:
> _Let me just create this user 😜_

https://docs.github.com/en/rest/users?apiVersion=2022-11-28

Reply:
> I pretty much mean the entire file. (All functions which are interacting directly with GitHub.) Unit tests should not have any external dependencies (GitHub). Integration tests are more extensive and also would show whether different systems (Shortinette and GitHub) are working together.

Thats true.
I also think that it would be a good idea to extract certain tests into separat folder.
We should also discuss this in our meeting this weekend aswell.